### DISCUSSION TITLE
Use KafkaBridge with KafkaUser

### DISCUSSION THREAD

**Discussion by nautiam** [strimzi/strimzi-kafka-operator#5947] [Q&A]
Hi all,

I setup a Kafka cluster with SASL_SSL authentication. And I created multiple KafkaUser with different permission on different topics. Now I want to create a KafkaBridge and limit the permission same as KafkaUser, so that when users call REST API through KafkaBridge, they have to add user/pass defined in KafkaUser and only have permission as ACLs in KafkaUser.
For example:
```
KafkaUser 1: user1 / password1
KafkaUser 2: user2 / password2
KafkaBridge url: http://bridge.example.com

User 1 call REST API: http://user1:password1@bridge.example.com
User 2 call REST API: http://user2:password2@bridge.example.com
```

Is there any way to do it?

Thanks in advance.

**Comment by scholzj** [comment]
No, we do not support this. The KafkaUser works only in the Kafka brokers and is used only between the Bridge and the Broker. To secure the HTP interface, you would need to use some API Gateway for example.

**Reply by nautiam** [reply]
Thanks for your answer. Can you give me any idea about which API Gateway and how can I make it work with Bridge to limit the access of user?

**Reply by scholzj** [reply]
Sorry, not really my area of expertise. So I'm not sure I can give any recommendations. The HTTP Bridge provides just a standard HTTP API. So any should work.
